src/components/MainPage.js

```diff
@@ -55,17 +63,20 @@ class MainPage extends Component {
 			If there is an error, error message is shown with the state updated to reflect this.
 	*/
   handleExchangeSearch = (crypto, intl) => {
+  	this.setState({ loading: true });
     CryptoAPICalls.exchangeSearch(this.state.selectedCrypto, this.state.selectedIntl)
 			.then(res => {
       if (res.data.length === 1) {
 					this.setState({
 						showingExchangeRate: true,
-						exchangeRate: res.data[0].ticker.ask
+						exchangeRate: res.data[0].ticker.ask,
+						loading: false
```

Comment:
This and the  `loading: false` on line 79 turn off the `LoadingSpinner` that shows while the comparison data is being fetched in both a success and failure situation.